packages/db/src/types.ts

```
@@ -111,84 +106,60 @@ type GetMinMaxJoin2<T> = (
-  update: Update<T>
-  updateAnd: UpdateAnd<T>
-  deleteEqual: DeleteEqual<T>
-  deleteEqualAnd: DeleteEqualAnd<T>
-  deleteLowerThan: DeleteLowerThan<T>
-  deleteWhere: DeleteWhere<T>
-  getTableColumns: GetTableColumns<T>
-  addColumn: AddColumn<T>
-  ensureColumns: EnsureColumns<T>
-  close: () => void
+  ready: Promise<void>;
+  createDatabases: (conf: DatabaseConfig, ...args: any) => Promise<void>;
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)